manifests/0000_50_cluster_monitoring_operator_04-config.yaml

```diff
@@ -186,6 +186,13 @@ data:
     # (monitoring-team) cluster:telemetry_selected_series:count tracks the total number of series
     # sent via telemetry from each cluster.
     - '{__name__="cluster:telemetry_selected_series:count"}'
+    # (rhmi) rhmi_version reports what version of RHMI is running on the cluster.
+    # Used to identify what versions are on a cluster that is experiencing problems.
+    - '{__name__="rhmi_version"}'
+    # (rhmi) rhmi_status reports the status of RHMI (pending, failed, complete)
```

Comment:
Could you please also add `@openshift/openshift-team-cluster-manager` in here for change notifications, as this metric will be consumed by OCM (for self-service RHMI)

Reply:
Thanks! :+1: